CMakeLists.txt

```diff
@@ -124,8 +114,7 @@ if(WITH_DAGMC)
       set(DAGMC_ROOT "${DEPS_ROOT_DIR}")
     endif()
     MESSAGE("-- DAGMC Root: ${DAGMC_ROOT}")
-    find_package(DAGMC)
-    message("-- DAGMC Found: ${DAGMC_FOUND}")
+    find_package(DAGMC REQUIRED)
```

Comment:
DAGMC is not required for PyNE

Reply:
Same logic here too, a variable `WITH_DAGMC` is available. Let me know.